sandy/njoy.py

```diff
@@ -855,7 +927,14 @@ def _groupr_input(endfin, pendfin, gendfout, mat,
         pk = len(ep) - 1
         text += [f"{pk} /"]
         text += [" ".join(map("{:.5e}".format, ep)) + " /"]
-    text += ["3/"]
+    if kwargs.get("mfcov", 1) == 40:
+        text += ["10/"]
+    else:
+        text += ["3/"]
+    if kwargs.get("mfcov", 1) == 34:
```

Comment:
mfcov is not a good  variable name because we are not handling covariances. 
Also  we should not use 34, 35 and 40 for the same reason (it should be 4, 5 and 10).

Also, since they are not exclusive,  there should be one kwarg per parameter, example
 - `mubar=False`
 - `pfns=False` or `chi=False`

Reply:
And for 40 (radioactive nuclide production)?

Reply:
not needed for the moment, I would just skip it!